WordPress/gutenberg
# Fix jest process hanging

## Description
This PR fixes jest e2e tests hanging definitely:

```
$ npx wp-scripts test-e2e --config=./packages/e2e-tests/jest.config.js packages/e2e-tests/specs/experiments/navigation.test.js 
```

## How has this been tested?
* Run the command above before applying this PR, confirm Jest process keeps running after all tests are passed
* Run the command above after applying this PR, confirm Jest process finishes properly.
* Hanging is somewhat non-deterministic, sometimes the tests work properly. To be extra sure run jest with `--clearCache` and retry a few times.
